### ISSUE TITLE
create an accompanying video

### ISSUE THREAD

**Issue by jsms90** [foundersandcoders/express-workshop#1]
In the same way that I asked @oliverjam to do for the TDD code-along - https://github.com/foundersandcoders/master-reference/issues/330, could you please create a youtube video of you completing this code along?

This seems to be a much more effective way of helping mentors who will be delivering this after you, than a written script. It's also really useful for the students themselves to be able to go back to this as a resource. See the [SGC notes from FAC11](https://github.com/foundersandcoders/london-curriculum/blob/e4c4d3cf1c0259dc5740b375616ff0b851311567/stop-go-continue/fac-11/cohort-week-6.md):
> Rachael: people do things at different speeds, so maybe we should have written inscructions/ a video you can go back to.
> ... Oli made a video. that's why it worked. Others really liked the video. Yahia: video takes off the pressure of typing along.

It also has the added benefit of helping you to prepare.

**Comment by PiotrBerebecki** [comment]
Yes, video would be really great 👍  Is there anyone in the space who can record this?
I'm away on Friday-Monday so I won't be able to complete this in time for next week.

**Comment by jsms90** [comment]
@PiotrBerebecki that's fine. If @Akin909 is the one delivering it, he needs to be the one practicing it anyway.

**Comment by jsms90** [comment]
@Akin909 A video is especially necessary, since there's only starter code in this repo. There's no completed version for students to look back to.

Given that this is a code along rather than a workshop, and students aren't being challenged to complete the tasks, I'd say it's especially necessary that we provide the completed version.

Other mentors have made a "mentor notes" folder, but this essentially amounts to writing a script for what a mentor is supposed to say. I would advise a video instead for the reasons suggested in the top comment.

**Comment by PiotrBerebecki** [comment]
Hi @jsms90, I've just double-checked and we have created a solution branch which includes the completed versions of all the tasks. Along with comments and express docs, it should be very beginner-friendly. But it does require a mentor to do some prep. For example, I needed probably a week to go through a number of tutorials to get a better understanding of Express. Then, around 3 -4 days to write up this workshop.

**Comment by jsms90** [comment]
Worst case scenario, we could just film @Akin909 while he's delivering it on Monday.

As for the solutions branch, [we're trying to get away from that pattern](https://github.com/foundersandcoders/express-handlebars-workshop/issues/1) because it's much more difficult to maintain.
